solutions/CSES Path Queries.mdx

```diff
@@ -2,273 +2,130 @@
 id: cses-1138
 source: CSES
 title: Path Queries
-author: Benjamin Qi
+author: Benjamin Qi, Neo Wang
 ---
 
 **Explanation:** [CPH 18.2](/CPH.pdf#page=174)
 
-**Time Complexity:** $(N+Q)\log N$
+Unlike the above explanation, instead of storing the subtree size, we can instead perform an Euler Tour in order to determine the bounds of each node's subtree. 
```

Comment:
The only difference is that you store the right index of the subtree instead of the subtree size. You make it sound like the above explanation doesn't involve performing an Euler tour?